PoopyExtensions/Extensions.cs

```diff
@@ -26,6 +26,107 @@ public static string Poopy(this string str)
             return Poopify(str.Select(s => string.Join(" ", str.Take(str.Count() - x++))));
         }
 
+        public static int PoopyKaprekar(this int num, int depth)
```

Comment:
Can you add a comment on this one because I honestly don't remember what the fuck the objective is